canonical/ubuntu.com
# [WD-10549] remove circled list item styles from tutorials

## Done

- As stated in the comments of styles.scss where the bug is coming from, Vanilla 4.0 does not support circled steps. Therefore used standard stepped list styles. The difference is in the dot that is now shown next to a number, which doesn't look attractive, but nevertheless fixes the bug.

## QA

Open any tutorial at /tutorials and check if the list of items show stepped list. 
On production all items are currently 0.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10549

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
